master/internal/job/job.go

```diff
@@ -5,14 +5,15 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/shopspring/decimal"
+	"golang.org/x/exp/slices"
 
-	"github.com/pkg/errors"
+	"github.com/shopspring/decimal"
 
 	"github.com/determined-ai/determined/master/pkg/actor"
 	"github.com/determined-ai/determined/master/pkg/model"
 	"github.com/determined-ai/determined/proto/pkg/apiv1"
 	"github.com/determined-ai/determined/proto/pkg/jobv1"
+	"github.com/pkg/errors"
```

Comment:
only thing new is the addition of standard library `slices` package, not sure why the rest got reordered. new gofumpt formatter?